photon-ml/src/main/scala/com/linkedin/photon/ml/optimization/GeneralizedLinearOptimizationProblem.scala

```diff
@@ -237,4 +261,20 @@ object GeneralizedLinearOptimizationProblem {
     val coefficients = model.coefficients.means
     coefficients.dot(coefficients) * regularizationWeight / 2
   }
+
+  /**
+    * Compute the Elastic Net regularization term value
+    *
+    * @param model the model
+    * @param regularizationWeight the weight of the regularization value
+    * @param regularizationContext the regularization context
+    * @return Elastic Net regularization term value
+    */
+  protected[ml] def getElasticNetRegularizationTermValue(model: GeneralizedLinearModel, regularizationWeight: Double,
+                                                         regularizationContext: RegularizationContext): Double = {
+
```

Comment:
For method declarations, use 4 space indentation for its parameters when they don't fit in a single line.

For more details, please take a look at [Spark's Scala code style guide](https://github.com/databricks/scala-style-guide#indent).